beta/.env.development

```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_HJ_SITE_ID = 2411683
-NEXT_PUBLIC_HJ_SITE_V = 6
\ No newline at end of file
+NEXT_PUBLIC_HJ_SITE_V = 6
+NEXT_PUBLIC_GA_TRACKING_ID = 'G-3918VS75Y6'
```

Comment:
Since we are going to use the same tracking ID in production and development we shall rather have it n the `next.config.js` alone?

Reply:
We should probably remove it from the local env after we're sure it's working as anticipated so we aren't tracking development work traffic